manifests/t/ThaUnknown/Miru/5.2.2/ThaUnknown.Miru.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ThaUnknown.Miru
+PackageVersion: 5.2.2
+InstallerType: nullsoft
+Scope: machine

```

Comment:
```suggestion
Scope: user
```
The installer installs to `%LOCALAPPDATA%` and puts its registry uninstall entries to `HKEY_CURRENT_USER`, making it a user-scoped install. This value has been incorrectly used in previous manifests as well, which essentially blocks winget users from upgrading this package correctly. If I install an older version (say `5.2.0`) and try to upgrade to winget's current latest `5.2.1`, it fails. WinGet detects the current install as user-scoped but the manifest for 5.2.1 says that it's a machine scoped install, making WinGet think that the upgrade is incompatible. See -

![image](https://github.com/user-attachments/assets/682aa506-de93-4fd8-a59e-c1cb5951227d)

Reply:
I honestly can't figure out why this happens for my life, f automation